docs/how-to/installation/kubernetes.txt

```diff
@@ -270,6 +270,22 @@ perform TLS termination in the load-balancer:
                serviceName: determined-master-service-<name for your deployment>
                servicePort: masterPort configured in values.yaml
 
+Default Scheduler (Optional)
+============================
+
+Determined includes support for the `lightweight coscheduling plugin
+<https://github.com/kubernetes-sigs/scheduler-plugins/tree/release-1.18/pkg/coscheduling>`__,
+which extends the default Kubernetes scheduler to provide gang
+scheduling. This feature is currently in beta and is not enabled by
```

Comment:
question: What do we mean by "beta" here? Are we saying it's untested/unverified, or that we may drop support for it in the future, or something else?

Reply:
I guess the closest definition would be "untested". While we have tested it a lot, I wouldn't be surprised if users found more undesirable behavior.

Reply:
Okay, I think this is probably fine then.